firmware/spade/README.md

```diff
@@ -30,14 +30,15 @@ Prerequisites:
 - The ability to run Bash scripts.
 - A C build environment, preferably Clang. On Windows, GCC won't work and you must use Clang. Make sure CMake and Make are both working.
```

Comment:
Just so you know - I want to delete these manual instructions entirely and push people down the path of using docker, such that we have *one* consistent build process...  beyond that, we even want to move people away from running these commands themselves - thus our focus recently on gardenshed/scripting